test/services/jobs/notifications/notifications-status-updates.service.test.js

```diff
@@ -37,6 +39,10 @@ describe('Job - Notifications - Process notifications status updates service', (
     // correctly, we do not want increase the timeout for the test as we want them to fail if a timeout occurs
     Sinon.stub(NotifyConfig, 'delay').value(ONE_HUNDRED_MILLISECONDS)
 
+    if (NotifyClient.prototype.getNotificationById.restore) {
+      NotifyClient.prototype.getNotificationById.restore()
+    }
```

Comment:
I'm guessing this is here for a reason, and it's related to `Sinon.restore()` not functioning as expected.

But it's a guess. Hence, this could really do with a `// NOTE: `.

Reply:
Yes. 

When we remove the notify stubs toggle this will be needed in all the tests. Unless we leave the functions as they where are remove the toggle.

Reply:
Cancel my last. 

It works without this. But i can not explain why it didn't before.